src/lib/features/playback-controls/ui/scrubbing-bar.svelte

```diff
@@ -23,10 +23,12 @@
     type="range"
     min={0}
     max={duration}
-    step={0.01}
+    step={0.001}
```

Comment:
question (non-blocking): why did you increase the precision? I'm concerned for keyboard users because a single left/right arrow press moves the slider value one step, so they will have to press for very long if they want to scrub. 

suggestion (non-blocking): I actually think even `0.01` is too fine of a precision, and we'd better stick to `0.1`.

Reply:
Because there is quite tangible snapping on drag end, but admittedly even after setting the step to a lower value it didn't go awaut.